packages/generator/src/sdk-metadata/sdk-metadata.ts

```diff
@@ -33,6 +33,17 @@ export async function sdkMetaDataJS(
   };
 }
 
+export async function sdkMetaDataJSFallback(
```

Comment:
question: Is this function used?
What would be the use case?

Reply:
not in the generator but it will be used in the metadata server app if we are asked for a service where we have no information. The reason I wanted to put it in the repo is a better code reuse.

Reply:
Hmm, but it really doesn't belong in the generator, right?
Would it be much less convenient if we put this into the metadata service?
What is the reuse potential?

Reply:
okay I also had not a good feeling about this. So I adjusted the method signature to take strings so that we can easily use them in the metadata server as well where we do not have generator options of service objects.